test/ufuzz.js

```diff
@@ -192,11 +187,12 @@ var ASSIGNMENTS = [
     '=',
     '=',
     '=',
+    '=',
+    '=',
+    '=',
+    '=',
+    '=',
 
-    '==',
-    '!=',
-    '===',
-    '!==',
     '+=',
```

Comment:
See if you like https://github.com/mishoo/UglifyJS2/pull/1926/commits/bc8fab5f041104663cc9e80b4dcfb720582cd004